.changeset/itchy-goats-joke.md

```diff
@@ -0,0 +1,10 @@
+---
+'@commercetools-frontend/application-shell': major
+---
+
+Remove the experimental render method `experimentalRenderAppWithRedux` from the `test-utils`.
+
+Instead, you can now pass to the `renderApp` and `renderAppWithRedux` methods the `disableApolloMocks` option. When this option is set to `true`, the real `ApolloProvider` is rendered instead of the Apollo `MockProvider`.
```

Comment:
```suggestion
Instead, you should pass the `disableApolloMocks` option to the `renderApp` and `renderAppWithRedux` methods. When this option is set to `true`, the real `ApolloProvider` is rendered instead of Apollo's `MockProvider`.
```